tests/generic_config_updater/test_mgmt_interface.py

```diff
@@ -0,0 +1,135 @@
+import ipaddress
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.generic_config_updater.gu_utils import apply_patch, create_path
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+from tests.route.test_forced_mgmt_route import FORCED_MGMT_ROUTE_PRIORITY, wait_for_file_changed
+
+pytestmark = [
+    pytest.mark.topology('any'),
+]
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def ensure_dut_readiness(duthost):
+    """
+    Setup/teardown fixture for pg headroom update test
+
+    Args:
+        duthost: DUT host object
+    """
+    create_checkpoint(duthost)
+
+    yield
+
+    try:
+        logger.info("Rolled back to original checkpoint")
+        rollback_or_reload(duthost)
+    finally:
+        delete_checkpoint(duthost)
+
+
+def update_forced_mgmt_route(duthost, interface_address, interface_key, routes):
+    # Escape '/' in interface key
+    json_patch = [
+        {
+            "path": create_path(["MGMT_INTERFACE",
+                                 "eth0|{}".format(interface_address.replace("/", "~1")),
```

Comment:
create_path should be able to handle the replace("/", "~1") part.

Reply:
Fixed